packages/amplify-category-notifications/src/__tests__/apns-cert-config.test.ts

```diff
@@ -21,14 +16,16 @@ describe('apns-cert-config', () => {
   const mockP12DecoderReturn = {};
 
   beforeAll(() => {
-    mockirer(inquirer, mockAnswers);
     const p12DecoderMock = p12decoder as jest.Mocked<typeof p12decoder>;
     p12DecoderMock.run.mockReturnValue(mockP12DecoderReturn as unknown as ICertificateInfo);
   });
 
-  beforeEach(() => {});
-
   test('p12decoder invoked', async () => {
+    prompter.input = jest
```

Comment:
Does the type checker yell at you when you do this?
I prefer this pattern which is a little less verbose and gives proper mock typings in the test:
```ts
import { prompter } from 'amplify-prompts';

jest.mock('amplify-prompts');
const prompterMock = prompter as jest.Mocked<typeof prompter>;

prompterMock.input.mockResolvedValueOnce(stuff).mockResolvedValueOnce(etc)
```